src/main/java/com/oltpbenchmark/api/collectors/DBParameterCollectorGen.java

```diff
@@ -30,6 +30,8 @@ public static DBParameterCollector getCollector(DatabaseType dbType, String dbUr
                 return new PostgresCollector(dbUrl, username, password);
             case COCKROACHDB:
                 return new CockroachCollector(dbUrl, username, password);
+            case YUGABYTE:
+                return new YuagbyteCollector(dbUrl, username, password);
```

Comment:
As discussed with Rakshit yesterday, we do not need  YugabyteCollector. We can use PostgresCollector for now. Let's not add code which is not needed.